src/lib/swapper/swappers/ThorchainSwapper/utils/addSlippageToMemo.ts

```diff
@@ -36,6 +36,10 @@ export const addSlippageToMemo = ({
     BigNumber.ROUND_DOWN,
   )
 
+  // Thorchain memo format:
+  // SWAP:ASSET:DESTADDR:LIM:AFFILIATE:FEE:DEX Aggregator Addr:Final Asset Addr:MinAmountOut
+  // Dex aggregator address, final asset address and minAmountOut are optional
+  // see https://gitlab.com/thorchain/thornode/-/merge_requests/2218 for reference
```

Comment:
Revert me, not applicable to this util since this isn't used by long tails (as you can see below, we only construct up to `affiliateBps` without the aggregator parts of the memo

Reply:
Well I added it because I felt like it was useful to also see the memo format there, there is one more line than the longtail version telling that the 3 last informations are optional

I removed it to follow your review, I don't really mind tbh as anyway we are testing it in unit tests now and there is an assert